packages/create-remix/templates/arc-stack/app/routes/index.tsx

```diff
@@ -0,0 +1,138 @@
+import type { ActionFunction, LoaderFunction } from "remix";
+import {
+  Form,
+  json,
+  redirect,
+  useActionData,
+  useLoaderData,
+  useLocation
+} from "remix";
+
+import { arc } from "~/db.server";
+import { getSession } from "~/session.server";
+
+import Alert from "@reach/alert";
+import { createNote, deleteNote } from "~/models/note";
+
+const loader: LoaderFunction = async ({ request }) => {
+  const session = await getSession(request);
+  const user = session.get("user");
+  if (!user) return redirect("/login");
+
+  const data = await arc.tables();
+  const result = await data.notes.query({
+    KeyConditionExpression: "sk = :sk",
+    ExpressionAttributeValues: { ":sk": `email#${user.email}` }
+  });
```

Comment:
This is what threw me for a loop when I noticed the `notes` table had reversed the `pk` and `sk` definitions. My gut reaction was "this is querying with only a sort key, this seems like a bug".

Reply:
thanks for taking a look at this 🙏